libs/signers/src/electron/index.ts

```
@@ -1,9 +1,9 @@
 import { getAndCacheSigner } from '../cache';
-import { ElectronSigner } from './signer';
 import type { AccountName } from '../names';
+import { ElectronSigner } from '@thecointech/electron-signer';
```

ESLint

[error] `@thecointech/electron-signer` import should occur before import of `../cache`

(import/order)